src/main/java/com/jit/studentmanagementsystem/controllers/CourseController.java

```diff
@@ -0,0 +1,72 @@
+package com.jit.studentmanagementsystem.controllers;
+
+import com.jit.studentmanagementsystem.dtos.CourseDTO;
+import com.jit.studentmanagementsystem.models.Course;
+import com.jit.studentmanagementsystem.services.CourseService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+import java.util.Map;
+
+@RestController
+@RequestMapping("/api/v1/courses")
+public class CourseController {
+
+    @Autowired
+    private CourseService courseService;
+
+    @GetMapping
+    //@ResponseBody
```

Comment:
RestController is Controller with additional ResponseBody

Reply:
So, I have to uncomment the ResponseBody?

Reply:
no, it is totally unnecessary, RestController includes it. just remove it and try not to have commented-out code

Reply:
ok